oxidation/libparsec/crates/client_types/src/logged_core.rs

```diff
@@ -0,0 +1,53 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 (eventually AGPL-3.0) 2016-present Scille SAS
+
+use once_cell::sync::OnceCell;
+use std::{
+    collections::{hash_map::DefaultHasher, HashMap},
+    hash::{Hash, Hasher},
+    sync::Mutex,
+};
+
+use libparsec_types::DeviceID;
+
+static CORE: OnceCell<Mutex<HashMap<i32, LoggedCore>>> = OnceCell::new();
+
+#[derive(Debug, Clone)]
+struct LoggedCore {
+    // TODO: Remove me when LoggedCore is implemented
+    test_device_id: DeviceID,
+}
+
+// We don't need to hash everything !
+impl Hash for LoggedCore {
+    fn hash<H: Hasher>(&self, state: &mut H) {
+        self.test_device_id.hash(state)
+    }
+}
+
+/// 32 bits of signature seems to be enough, people don't have that much devices
```

Comment:
I'm not super happy to have JavaScript stuff leaking into the pure rust :(

I think we should add a max check in the bindings to raise exception if the returned value is too big

This way we can still use whatever value we want here

Regarding for what value we want here, I'd rather use an unsigned value

Reply:
So what's the equivalent of uint in python ?
Because I have to add in api.py